app/notices/shop_employee_notices/employee_sep_qle_accept_notice.rb

```diff
@@ -0,0 +1,8 @@
+class ShopEmployeeNotices::EmployeeSepQleAcceptNotice < ShopEmployeeNotice
+
```

Comment:
created a class because in future if we want to add extra append to this class by adding methods.